reverse_proxy/templates/cran.conf.j2

```diff
@@ -0,0 +1,66 @@
+server {
+  listen 80;
+  server_name cran.{{zone}};
+
+  location /cran-fetch {
+    internal;
+    proxy_pass {{cran_mirror}};
+  }
+
+  location / {
+    # prevent cyclic location.capture
+    if ($request_uri ~ /cran-fetch) { break; }
```

Comment:
Is this needed? Shouldn't Nginx matching rules prevent this cycle?
https://www.digitalocean.com/community/tutorials/understanding-nginx-server-and-location-block-selection-algorithms#how-nginx-chooses-which-location-to-use-to-handle-requests

Reply:
I thought this too but it got stuck in a loop without it :sweat_smile: